src/platformdirs/windows.py

```diff
@@ -12,6 +12,15 @@
 if TYPE_CHECKING:
     from collections.abc import Callable
 
+try:  # noqa: SIM105
+    import ctypes
+except ImportError:
+    pass
+try:  # noqa: SIM105
+    import winreg
+except ImportError:
+    pass
```

Comment:
Upon taking a closer look, let's do with the rest of the file does and lazily import when necessary.

Reply:
The rest of the file does not do lazy importing at the moment because this pull request removes all lazy importing from `windows.py`. Are you saying that I should drop the “Simplify logic for choosing `get_win_folder`’s value” commit?